Add Configurable mounting steps to C2C Deployment guide (#6106)
ballerina-platform/ballerina-dev-website: swan-lake/learn-the-platform/run-in-the-cloud/code-to-cloud/code-to-cloud-deployment.md

cloud = "k8s" 
```

***Config.toml***

> Contains the values which needs to be passed into configurable variables. You can read the [Configurable learn page](https://ballerina.io/learn/configure-ballerina-programs/configure-a-sample-ballerina-service/) to learn more about this. When you are using the Cloud features in ballerina, Config.toml is not being packed with container image as it can contain sensitive information. You need to explicitly provide this file in the runtime. 

Reviewer: ```suggestion
Contains the values, which need to be passed into the configurable variables. To learn more about this, see [Configure a sample Ballerina service](https://ballerina.io/learn/configure-ballerina-programs/configure-a-sample-ballerina-service/). When you are using the cloud-related features in Ballerina, the `Config.toml` file is not being packed with the container image as it can contain sensitive information. You need to explicitly provide this file in the runtime. 
```

EXPOSE  9090
USER ballerina

> CMD java -Xdiag -cp "anjana-http2_test-0.1.0.jar:jars/*" 'anjana/http2_test/0/$_init'

Reviewer 2: Is it ok to have usernames?

Author: this is due to folder structure being different in my test setup. I'll change this in another PR ;)

Reviewer 2: Got it! No problem.

    wso2inc/hello                 v0.1.0              60d95f0928b2        About a minute ago   228MB
    ```

> 2. Execute the `docker run -d -v <path/to/config>/Config.toml:/home/ballerina/Config.toml -p 9090:9090 wso2inc/hello:v0.1.0` command to run the generated Docker image. You need to make sure to set the correct path of the created `Config.toml`. Using volume mount is just one way of providing the `Config.toml` file into the runtime. Other ways of loading providing Configurable values can be found on the [Configurable specification.](https://github.com/ballerina-platform/ballerina-spec/blob/master/configurable/spec.md#locating-toml-files)

Reviewer: ```suggestion
2. Execute the `docker run -d -v <path/to/config>/Config.toml:/home/ballerina/Config.toml -p 9090:9090 wso2inc/hello:v0.1.0` command to run the generated Docker image. 

>**Info:** Make sure to set the correct path of the created `Config.toml`. Using volume mount is one way of providing the `Config.toml` file into the runtime. For other ways of providing configurable values, see the [Ballerina specification](https://github.com/ballerina-platform/ballerina-spec/blob/master/configurable/spec.md#locating-toml-files).
```

        }
    }
    ```

> 3. Create `Config.toml` in the package directory and paste the following contents inside.

Reviewer: ```suggestion
3. Create the `Config.toml` file in the package directory and paste the following content into it.
```

    max_replicas=5 # Maximum number of replicas of the container can scale up to
    cpu=60 # Average CPU utilization of pods

>     [[cloud.config.files]] # Mounts Config.toml file as a Config Map in kubernetes

Reviewer: ```suggestion
    [[cloud.config.files]] # Mounts the `Config.toml` file as a config map in Kubernetes.
```
